druntime/posix.mak

```diff
@@ -350,8 +350,8 @@ $(IMPDIR)/%.h : src/%.h
 
 ######################## Build DMD if non-existent ##############################
 
-$(DMD):
-	$(MAKE) -C $(DMD_DIR)/src -f posix.mak BUILD=$(BUILD) OS=$(OS) MODEL=$(MODEL)
+$(DMD_DIR)/../generated/$(OS)/$(BUILD)/$(MODEL)/dmd:
```

Comment:
This is a subtle but important change. E.g., for CI we specify a `DMD` var explicitly in the druntime make command-line (in some cases at least). Don't pretend that we are able to generate *that* DMD (in case it's non-existent) with the following recipe - the output path is fixed.